test/functional/editions_controller_test.rb

```diff
@@ -64,4 +64,55 @@ class EditionsControllerTest < ActionController::TestCase
       assert_equal EditionsController.new.method(:metadata).super_method.name, :show
     end
   end
+
+  context "when 'restrict_access_by_org' feature toggle is disabled" do
```

Comment:
instead of disabled, can we use false. as 'false' best describes the value of the switch

Reply:
I think enabled/disabled is common parlance for feature toggle status, and reads better than true/false.